app/controllers/supported_demands_controller.rb

```diff
@@ -23,6 +23,12 @@ def update
     end
   end
 
+  def destroy
+    @demand = find_demand
+    current_user.user_demands.find_by(demand: @demand).destroy
```

Comment:
Could make this a little safer with:
```
current_user.user_demands.find_by(demand: @demand)&.destroy
```
so that it won't explode if the demand isn't found.